mne/forward/forward.py

```diff
@@ -1078,23 +1080,50 @@ def compute_depth_prior(G, gain_info, is_fixed_ori, exp=0.8, limit=10.0,
     is_fixed_ori : bool
         Whether or not ``G`` is fixed orientation.
     exp : float
-        Exponent for the depth weighting.
+        Exponent for the depth weighting, must be between 0 and 1.
     limit : float | None
-        The upper bound on depth weighting. Can be None to be unbounded.
+        The upper bound on depth weighting.
+        Can be None to be bounded by the largest finite prior.
     patch_areas : ndarray | None
         Patch areas of the vertices from the forward solution.
-    limit_depth_chs : bool
-        If True, limit depth weighting computation to gradiometers
-        if present (ignoring magnetometers). This can improve the
-        computations on systems that have both (e.g., Neuromag).
-    loose_method : 'svd' | 'sum'
+    limit_depth_chs : bool | 'whiten'
+        How to deal with multiple channel types in depth weighting. Options:
+
+        :data:`python:True` (default)
+            Use only grad channels in depth weighting (equivalent to MNE C
+            minimum-norm code). If grad channels aren't present, only mag
+            channels will be used (if no mag, then eeg). This makes the depth
+            prior dependent only on the sensor geometry (and relationship
+            to the sources).
+        ``'whiten'``
+            Compute a whitener and apply it to the channels before computing
+            the depth prior. In this case ``noise_cov`` must not be None.
+
+            .. versionadded:: 0.18
+        :data:`python:False`
+            Use all channels. Only recommended when the gain matrix has
+            already been whitened (otherwise it will be arbitrarily
+            biased toward whichever channel type has the largest values in
+            SI units). Whitening the gain matrix makes the depth prior
+            depend on both sensor geometry and the data of interest captured
+            by the noise covariance (e.g., projections, SNR).
+
+    combine_xyz : 'spectral' | 'L2'
         When a loose (or free) orientation is used, how the depth weighting
         for each triplet should be calculated.
-        If 'svd', use the largest singular value of the 3x3 matrix
-        formed by the dot product ``Gk.T @ Gk`` will be used.
-        If 'sum', then ``np.sum(Gk ** 2)`` will be used.
+        If 'spectral', use the squared spectral norm of Gk.
+        If 'L2', use the squared L2 norm of Gk.
```

Comment:
how about we call it 'fro' as it's the frobenius norm of the block of 3 cols. Like used in linalg.norm?